package.json

```diff
@@ -119,7 +128,13 @@
         }
       ]
     },
+    "prettier": {
+      "arrowParens": "avoid"
+    },
     "eslint": {
+      "rules": {
+        "arrow-parens": ["warn", "as-needed"]
+      },
```

Comment:
Prettier[ changed the default](https://prettier.io/docs/en/options.html#arrow-function-parentheses) of `arrow-parens` from `avoid` to `always` in v2.0.0. Keep it as "avoid" to avoid unnecessary diffs. We can decide whether to accept the new default later.